tests/Fixer/Basic/Psr4FixerTest.php

```diff
@@ -29,7 +29,7 @@ public function testFixCase()
         $fileProphecy->willExtend(\SplFileInfo::class);
         $fileProphecy->getBasename('.php')->willReturn('Bar');
         $fileProphecy->getExtension()->willReturn('php');
-        $fileProphecy->getRealPath()->willReturn(__DIR__.'/Psr4/Foo/Bar.php');
+        $fileProphecy->getRealPath()->willReturn(__DIR__.\DIRECTORY_SEPARATOR.'Psr4'.\DIRECTORY_SEPARATOR.'Foo'.\DIRECTORY_SEPARATOR.'Bar.php');
```

Comment:
is this change actually needed?
we were using `/` everywhere, and PHP under the hood were always playing fair for us, allowing for `/`and `\`as dir separator, and silence conversion under the hood for Windows paths, if needed

Reply:
it's needed, I'm using `realpath` function in the fixer as there are 2 paths to compare - the path of the file being fixed and path given as `dir` value in config.